openmeetings-web/src/main/java/org/apache/openmeetings/web/admin/connection/ConnectionsPanel.java

```diff
@@ -57,9 +57,9 @@
 	@SpringBean
 	private ClientManager cm;
```

Comment:
`private IClientManager cm;`

Should be probably also IClientManager

Reply:
why?

Reply:
We created the interface IClientManager to make sure we use the right kind of API methods to access the clientManager (and to potentially change the client manager into another storage option, like Redis or sth - something that doesn't require increasing memory with number of users!) 

But if we then wire up the concrete class instead of the Interface, that will break it! You can't swap ClientManager with a different implementation.

Reply:
`IClientManager` was created since we split our application to modules
To allow access to some client manager features from outside `openmeetings-web`

this is HACK :(

all these interfaces are useless ...